django/librarian/models.py

```diff
@@ -344,6 +348,22 @@ class Meta:
     # may be uploaded under different filenames
     filename = models.CharField(max_length=255, null=True, blank=True)
 
+    # Specific to PDF documents.
+    # The extraction method *that was used* to extract text from the PDF
+    pdf_extraction_method = models.CharField(
+        max_length=40, null=True, blank=True, choices=PDF_EXTRACTION_CHOICES
+    )
+    # Page range to extract from PDF (e.g. "1-3, 5, 7-9")
+    pdf_page_range = models.CharField(max_length=255, null=True, blank=True)
+    # Crop box for PDF extraction (left, right, top, bottom) - ratio of page size
+    pdf_crop_left = models.FloatField(null=True, blank=True)
+    pdf_crop_right = models.FloatField(null=True, blank=True)
+    pdf_crop_top = models.FloatField(null=True, blank=True)
+    pdf_crop_bottom = models.FloatField(null=True, blank=True)
+    # The number of detected / extracted pages in the PDF (for debugging / confirmation)
+    pdf_num_pages_detected = models.IntegerField(null=True, blank=True)
+    pdf_num_pages_extracted = models.IntegerField(null=True, blank=True)
```

Comment:
We aren't using all the fields in the model right now. I am pushing off the work on PDF crop and page ranges to a later PR.